### ISSUE TITLE
I can't use my mouse to click on the osc anymore if this is in my scripts.

### ISSUE THREAD

**Issue by ChocoBar9** [ekisu/mpv-webm#58]
ver. 2.0.0.0 (mpv-x86_64-20181230-git-d6d6da4) Win10

Okay since I've finally figured out how to do the log file command I can ask for a proper diagnostics. I recently added your script to my setup and it works and everything but the problem is now I can't access my osc with my mouse anymore, it pops up but I can't move the cursor or pause and play it. 
[log.txt](https://github.com/ekisu/mpv-webm/files/3333780/log.txt)

**Comment by ekisu** [comment]
Okay, there are a lot of weird things in that log:
- a lot of scripts are seemingly loaded twice (or more times), as you can see in the beginning of the log (`webm` and `webm2`; `patched_osc` and `patched_osc2`; `stats`, `stats2` and `stats3`; there's a lot more of them). Most of them seem to load from the same script path, but one with forward-slashes and one with back-slashes.
```
[   0.015][d][webm] Loading lua script C:/Users/Choco/AppData/Roaming/mpv/scripts/webm.lua...
[   0.022][d][webm2] Loading lua script C:\Users\Choco\AppData\Roaming/mpv/scripts/webm.lua...
```
- very early in the log, both `patched_osc` and `patched_osc2` crash with the same error:
```
patched_osc.lua:86: attempt to index global 'Thumbnailer' (a nil value)
```

You also have two `mpv_thumbnail_script_client_osc` scripts running, which I'm pretty sure isn't correct. Sincerely I have no idea why this is happening, and if/how my script even causes this in the first place.

Do you mind sending a screenshot of your scripts folder? Also try removing some of the scripts, it might help pinpointing the issue.

**Comment by ChocoBar9** [comment]
> 
> 
> Okay, there are a lot of weird things in that log:
> 
>     * a lot of scripts are seemingly loaded twice (or more times), as you can see in the beginning of the log (`webm` and `webm2`; `patched_osc` and `patched_osc2`; `stats`, `stats2` and `stats3`; there's a lot more of them). Most of them seem to load from the same script path, but one with forward-slashes and one with back-slashes.
> 
> 
> ```
> [   0.015][d][webm] Loading lua script C:/Users/Choco/AppData/Roaming/mpv/scripts/webm.lua...
> [   0.022][d][webm2] Loading lua script C:\Users\Choco\AppData\Roaming/mpv/scripts/webm.lua...
> ```
> 
>     * very early in the log, both `patched_osc` and `patched_osc2` crash with the same error:
> 
> 
> ```
> patched_osc.lua:86: attempt to index global 'Thumbnailer' (a nil value)
> ```
> 
> You also have two `mpv_thumbnail_script_client_osc` scripts running, which I'm pretty sure isn't correct. Sincerely I have no idea why this is happening, and if/how my script even causes this in the first place.
> 
> Do you mind sending a screenshot of your scripts folder? Also try removing some of the scripts, it might help pinpointing the issue.

![mpv](https://user-images.githubusercontent.com/7196448/60289386-40e18e00-98e4-11e9-9f16-413f26f6bec3.PNG)

**Comment by ekisu** [comment]
Well, that's a lot of scripts. The OSC you're currently using is the `mpv_thumbnail_script` one, right? Does it work if you only enable that script (client and server) and the webm script?

**Comment by ChocoBar9** [comment]
> 
> 
> Well, that's a lot of scripts. The OSC you're currently using is the `mpv_thumbnail_script` one, right? Does it work if you only enable that script (client and server) and the webm script?

Yes. The speed of how many thumbnails are uploaded depends on how many copies of the server is in the folder. I tried a new scripts folder with only that the webm and the osc and patched osc files and it works.

**Comment by ekisu** [comment]
That means some other script is to blame for this. I guess now it's trial and error, to find out which script breaks everything.
